cmd/funcbench/go.mod

```diff
@@ -0,0 +1,13 @@
+module funcbench
```

Comment:
unfortunately we can't have separate go.mod files for each tool as it messes with testing building etc.

run go build cmd/funcbench/main.go from the repo root and it will add the dependencies to the root go.mod file.

And after that need to remoe go.mod and go.sum from cmd/funcbench